testingCommands.txt

```diff
@@ -0,0 +1,5 @@
+FD 100 RT 90 FD 100 RT 90 FD 100 RT 90 FD 100
+REPEAT 360 [lt 1 fd 1]
+REPEAT 36 [REPEAT 360 [lt 1 fd 1] RT 10]
+FD 100 PU FD 100 PD FD 100
+SETC 1 SETPN 1 SETPC 1 1
```

Comment:
This should be in git?

Reply:
Yes, for ginder head and another person in team